crates/api_models/src/api_keys.rs

```diff
@@ -136,8 +136,8 @@ pub struct UpdateApiKeyRequest {
 pub struct RevokeApiKeyResponse {
     /// The identifier for the API Key.
     #[schema(max_length = 64, example = "5hEEqkgJUyuxgSKGArHA4mWSnX")]
+    pub merchant_id: String,
```

Comment:
please correct the schema here. I believe this `/// The identifier for the API Key.
    #[schema(max_length = 64, example = "5hEEqkgJUyuxgSKGArHA4mWSnX")]` doesn't describe `merchant_id`

Reply:
@Chethan-rao , @SanchithHegde should merchant_id, key_id be added as primary key here ?
https://github.com/juspay/hyperswitch/blob/f624eb52d61561c365cce21e58b08281d096d904/crates/storage_models/src/api_keys.rs#L7

Reply:
No, key_id is enough to uniquely identify records